guides/configure-oE-devEnviron-ubuntu-vm/How-to-install-openEQUELLA-Development-Environment-in-Virtual-Box-Ubuntu-OS.md

```diff
@@ -0,0 +1,626 @@
+
+1. Ask for access to the BYUI openEquella repository in GitHub or fork your own branch.
+
+2. Generate a new SSH key (this is necessary to check the code in and out from GitHub. Open a terminal and run the command below.
+
+`$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"`
+
+output: `Generating public/private rsa key pair.`
+
+3. When you're prompted to "Enter a file in which to save the key," press Enter. This accepts the default file location.
+
+`> Enter a file in which to save the key (/home/you/.ssh/id_rsa): [Press enter]`
+
+4. At the prompt, type a secure passphrase. For more information, [see "Working with SSH key passphrases".](https://docs.github.com/en/articles/working-with-ssh-key-passphrases)
+
+`> Enter passphrase (empty for no passphrase): [Type a passphrase]`
+`> Enter same passphrase again: [Type passphrase again]`
+
+5. Now you just add the SSH key to the ssh-agent. To do this start the ssh agent in the background.
+
+`$ eval "$(ssh-agent -s)"`
+`> Agent pid 59566`
+
+6. Now add the SSH private key to the ssh-agent. Use the same name that you specified in step four (id_rsa)
+
+`$ ssh-add ~/.ssh/id_rsa`</br>
+`> Enter passphrase for /home/developer/.ssh/id_rsa`
+
+### <a id="user-content-add-key" class="anchor" aria-hidden="true" href="#add-key"></a>Add the ssh key to your openEQUELLA GitHub Repository
+
+1. Navigate to your .ssh directory and type the following:
+
+`$ cat id_rsa.pub`
+
+![copy key](images/github/02%20copy%20key.png)
+
+2. Highlight the ssh key, right click and select “copy”.
+
+![copy key2](images/github/03%20copy%20key2.png)
+
+3. Login to your GitHub account and go to the openEQUELLA GitHub repository. Then right click on the profile icon in the upper right hand corner of the screen and select “Settings”
+
+![settings](images/github/01%20settings.png)
+
+4. Click on SSH and GPG keys.
+
+![copy key2](images/github/04%20copy%20key2.png)
+
+5. Click on the “New SSH Key” button.
+
+![new key](images/github/06%20new%20key.png)
+
+6. Give the ssh key a name, and past the key into the Key field. Then click the “Add SSH key” button.
+
+![paste key](images/github/07%20paste%20key.png)
+
+### <a id="user-content-clone-repo" class="anchor" aria-hidden="true" href="#clone-repo"></a>Clone the openEQUELLA Repository
+
+1. Navigate to the home page of the openEQUELLA repository and click on the green “Code” button. Then copy the git command to “Clone with SSH”.
+
+![get code](images/clone/01%20get%20code.png)
+
+2. From the command line:
+a. Navigate to your home directory.
+b. Create a directory where you will store your git repositories
+c. Configure your git name and email
+d. Clone the openEquella repository
+
+`$ cd /home/developer`</br>
+`$ mkdir git_proj`</br>
+`$ git config --global user.name “mjm”`</br>
+`$ git config --global user.email someones@gmail.com`</br>
+`$ git clone git@github.com:someone/openEQUELLA.git `
```

Comment:
same as above
https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks